_spinnaker_install_admin_guides/halyard-gitops.md

```diff
@@ -132,7 +128,7 @@ In Github, navigate to the settings of the repository containing your Spinnaker
 
 Note: You'll need to make sure that the service is exposed externally and reachable by [Github servers](https://help.github.com/en/articles/about-githubs-ip-addresses).
 
-## Option 2: Use CI
+### Option 2b: Use CI
```

Comment:
can you change `DEPLOYMENT_NAME` to something else? It collides with the concept of halyard deployment environments, along with your recommendation above where you say:
> you should also not store multiple deployments in the same `config` file....